source/_integrations/zwave_js.markdown

```diff
@@ -339,6 +339,21 @@ These are notification events fired by devices using the Entry Control command c
 }
 ```
 
+#### `Automation example`
```

Comment:
I think we should put this section directly within the higher level `Node events (Notification)` block above without a new separate sub header. That is more in line with how the other event sections are written.

Reply:
Do you mean just above the `Notification Command Class` heading?

Reply:
Yes, exactly. 👍

Reply:
Done!